Feature bjs2 41816

ahim-miha opened the pull request:
Тут я крч чет опять не то сделал и у меня показывает изменения в тех файлах которые я не трогал, все шо с LikePost связано мое(метод в контроллере и реализация в сервисе и две дто)

Есть какие то конфликты, не особо понимаю с чем они связаны и как их решить и я чет не то сделал походу, в измененных файлах числяться те которые я не трогал

kraiush committed:
BJS2-27289: Analysis comments V2.0

mikhailov26 committed:
Merge pull request #1105 from CorporationX/werewolf-kraiu

BJS2-27289: Analysis comments V2.0

ahim-miha committed:
https://faang-school.atlassian.net/browse/BJS2-41816

Уведомление о лайке

BobrAll commented:
Привет, сможешь отдельным pr перезакинуть решение? А то тут 60 файлов и куча конфликтов, я думаю и тебе не очень комфортно будет их фиксить, и мне среди всех этих файлов искать изменения :)
Можешь потом ссылку кинуть в чат дискорда и меня тегнуть

ahim-miha committed:
https://faang-school.atlassian.net/browse/BJS2-41816

Вроде все нужное, как отправить по отдельности так и не понял, просто по итогу немного улучшил шо бы отправить

BobrAll commented:
> Привет, сможешь отдельным pr перезакинуть решение? А то тут 60 файлов и куча конфликтов, я думаю и тебе не очень комфортно будет их фиксить, и мне среди всех этих файлов искать изменения :) Можешь потом ссылку кинуть в чат дискорда и меня тегнуть

Нужны твои комментарии)

ahim-miha commented:
https://github.com/CorporationX/post_service/pull/1296/commits/2f8415fbf8005c91ce78fe063c69dc669d45d3a9
Этот коммит с задачей связанные

ahim-miha commented:
Конфликты решу следующим коммито как обновления подтяну с репозитория, там полная ж если честно, только в этом случае ж в кубе, так что это еще больше чем надолго
